lib/prelude.dx

```diff
@@ -1398,7 +1398,7 @@ def fread (stream:Stream ReadMode) : {IO} String =
 '### Print
 
 def getOutputStream (_:Unit) : {IO} Stream WriteMode =
-  MkStream $ %ptrLoad OUT_STREAM_PTR
+  MkStream $ %ptrLoad %outputStreamPtr
```

Comment:
huh, is that legal? I thought that arguments to builtins can only be variables

Reply:
Oh, good point. Maybe that changed at some point. Currently, it looks like we just have to be able to infer the types of the arguments without using any information about the type of the builtin. Bad things probably happen if that fails.